source/guides/contributing/building.md

```diff
@@ -0,0 +1,28 @@
+Working With and Building Denizen
+---------------------------------
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+### Prerequisites
+
+Make sure you have the following tools installed on your machine:
+
+- Some form of git. We recommend [GitHub Desktop](https://desktop.github.com/) <span class="parens">(if you're using Linux, don't worry, there's probably a package for your distro)</span>.
+- [IntelliJ IDEA](https://www.jetbrains.com/idea/). This editor is specifically designed for Java projects and will handle dependencies on its own.
+
+### Building Spigot
+
+To build Denizen, you'll also need to build the supported versions of Spigot. These versions are 1.16.5, 1.17.1, and 1.18.1. To do this, Spigot provides a tool called [BuildTools](https://www.spigotmc.org/wiki/buildtools/).
```

Comment:
Don't hard-specify versions in documentation (as that means it will have to be updated every time minecraft updates). Should reference either the module poms or the github readme for applicable version list.